.drone.yml

```diff
@@ -330,3 +330,93 @@ trigger:
     - cron
   cron:
     - rebuild-devcenter-prod-cron
+
+---
+depends_on: null
+kind: pipeline
+type: kubernetes
+name: build-preview
+
+resources:
+  requests:
+    cpu: 8
+    memory: 4GiB
+
+steps:
+  - name: publish-preview # The ${number} var comes from the webhook query parameters.
+    image: plugins/kaniko-ecr
+    environment:
+      NPM_AUTH:
+        from_secret: NPM_AUTH
+      NPM_EMAIL:
+        from_secret: NPM_EMAIL
+      REALM_API_URL:
+        from_secret: REALM_API_URL
+      REALM_SEARCH_URL:
+        from_secret: REALM_SEARCH_URL
+      STRAPI_URL:
+        from_secret: STRAPI_URL_PROD
```

Comment:
Just a heads up, I recently merged changes to update this to use `STRAPI_URL_STAGING` (https://github.com/mongodb/devcenter/pull/238). I think for previews we want this as well, since we now have a way to refresh staging DB.